src/java/htsjdk/samtools/util/Locatable.java

```diff
@@ -0,0 +1,9 @@
+package htsjdk.samtools.util;
+
+public interface Locatable {
+    /**
+     * gets a {@link SimpleInterval} matching the mapping of this locatable to the reference genome
+     * @return a SimpleInterval or null if this is unmapped
+     */
+    SimpleInterval getInterval();
```

Comment:
Maybe call this ```getSimpleInterval``` since we have a class called ```Interval``` which presumably could be the inferred return type?